### ISSUE TITLE
Peer Dependencies Error

### ISSUE THREAD

**Issue by haydn** [uber/react-vis#70]
It looks like recent versions of `react-addons-shallow-compare` (15.x.x) cause peer dependency problems:

```
npm ERR! peerinvalid The package react@0.14.8 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-addons-shallow-compare@15.0.2 wants react@^15.0.2
npm ERR! peerinvalid Peer react-addons-test-utils@0.14.8 wants react@^0.14.8
npm ERR! peerinvalid Peer react-dom@0.14.8 wants react@^0.14.8
```

Dropping back to 0.14.8 works fine.

**Comment by chrisirhc** [comment]
@haydn looks like `"react-addons-shallow-compare"` should be in peerDependencies with a permissive version number, and in `devDependencies` with `>=0.14.0 <0.15.0`.

**Comment by bulyonov** [comment]
Closing this issue since #74 is now landed (please install react-vis 0.3.4 or later to use this change).
